src/components/tictactoe.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import Board from './JOCboard';
+import { css } from 'emotion';
+
+interface GameState {
+  history: Array<{
+    position?: { row: string; col: string };
+    squares: Array<string | null>;

```

Comment:
I suspect this string could instead be `'X' | 'O'`? If so, probably worth defining that as a type somewhere, e.g. `type FilledSpace = 'X' | 'O'`